Resources/config/routing.yml

```diff
@@ -14,3 +14,8 @@ template_yui_module:
 ez_pjax:
     resource: "@eZPlatformUIBundle/Resources/config/routing_pjax.yml"
     prefix:   /pjax
+
+yui_combo_loader:
+    path: /combo
```

Comment:
Should we go for something a bit less likely to collide ? If a content is named "Combo", we're gonna have issues :-)